src/components/facts/list/form.js

```diff
@@ -75,51 +73,54 @@ export class FactForm extends Component {
   onSubmit() { this.props.onSubmit(this.state.runningFact) }
 
   render() {
-    const buttonClasses = ['ui', 'button', ...(this.isValid() ? [] : ['disabled'])]
+    const buttonClasses = ['ui', 'button', 'tiny', 'primary', ...(this.isValid() ? [] : ['disabled'])]
     const {props: {buttonText}, state: {runningFact: {expression, name, variable_name}}} = this
 
     return (
+    <div className='Fact--outer'>
       <div className='Fact new ui form'>
-        <div className='row'>
-          <div className='col-md-3'>
-            <div className={`field ${this.isExpressionValid() ? '' : 'error'}`}>
+        <div className='section-simulation simulation-sample'>
+          <div className={`field ${this.isExpressionValid() ? '' : 'error'}`}>
+            <input
+              type='text'
+              placeholder='value'
+              value={expression}
+              onChange={this.onChangeExpression.bind(this)}
+              onBlur={this.onBlurExpression.bind(this)}
+            />
+          </div>
+        </div>
+        <div className='section-name'>
+          <div className='variable-name'>
+            <div className={`field ${this.isVariableNameUnique() ? '' : 'error'}`}>
+              <span className='prefix'>#</span>
               <input
                 type='text'
-                placeholder='Expression'
-                value={expression}
-                onChange={this.onChangeExpression.bind(this)}
-                onBlur={this.onBlurExpression.bind(this)}
+                placeholder='variable_name'
+                value={variable_name}
+                onChange={this.onChangeVariableName.bind(this)}
+                onKeyDown={(e) => {if (e.keyCode === 13 && this.isValid()) {this.onSubmit()}}}
```

Comment:
Not your doing, I know, but if you're editing this anyways probably best to pull this into a `submitIfEnter` helper or something (it is repeated below).